tests/context-info.sh

```diff
@@ -433,10 +433,11 @@ run_test "Binary file instead of text file" \
 
 # End of tests
 
+echo
```

Comment:
Why moving to `echo` here ? Just wondering if there is some specific reason.

Reply:
I find `echo` clearer than `printf` since you don't have to worry about escape sequences. They each have their downside: `echo` only works portably on strings that don't contain backslashes and don't start with a dash, but it's straightforward for those strings. `printf` is well-defined but you need to remember about its escapes (and it's slower in some shells where it isn't built in, but since it's built into dash, bash and zsh, that's not much of a concern in the context of this script).